cmd/ebrelayer/main.go

```diff
@@ -189,6 +194,32 @@ func initConfig(cmd *cobra.Command) error {
 	return viper.BindPFlag(flags.FlagChainID, cmd.PersistentFlags().Lookup(flags.FlagChainID))
 }
 
+func replayEthereumCmd() *cobra.Command {
+	//nolint:lll
+	replayEthereumCmd := &cobra.Command{
+		Use:     "replayEthereum [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker] [validatorMnemonic] [fromBlock] [toBlock]",
+		Short:   "replay missed ethereum events",
+		Args:    cobra.ExactArgs(9),
```

Comment:
You have comments for 7 arguments ([tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker] [validatorMnemonic] [fromBlock] [toBlock]), and you're requiring 9.

Reply:
will add the two arguments sifFromBlock and sifEndBlock

Reply:
Are we going to add any documentation about when to use this?  Do you have to go through the logs to figure out which blocks have been missed?  Can you run it more than once with the same start/end blocks?

Reply:
please see replay-instruction.md in docs/relayer. You can run it again, will generate some duplicated transactions. No other side effect.